networkit/cpp/community/PLM.cpp

```diff
@@ -51,7 +55,7 @@ void PLM::run() {
     std::vector<double> volCommunity(o, 0.0);
     zeta.parallelForEntries([&](node u, index C) { // set volume for all communities
         if (C != none)
-            volCommunity[C] = volNode[u];
+            volCommunity[C] += volNode[u];
```

Comment:
This needs to be an atomic increment as this in a parallel loop.

Reply:
Or rather an OpenMP reduction...?

Reply:
Good point, I updated the loop, same way as the loop on line 267, thank you !